ceno_zkvm/src/precompiles/uint256.rs

```diff
@@ -381,6 +380,260 @@ impl<E: ExtensionField> ProtocolWitnessGenerator<E> for Uint256MulLayout<E> {
     }
 }
 
+pub trait Uint256InvSpec {
+    type P: FieldParameters + NumWords;
+    fn syscall() -> u32;
+    fn name() -> String;
+    fn modulus() -> BigUint;
+}
+
+/// A set of columns for the Uint256Inv operation.
+#[derive(Debug, Clone, AlignedBorrow)]
+#[repr(C)]
+pub struct Uint256InvWitCols<T> {
+    // x = UInt256Field::ONE
+    // y := input
+    // y in little endian format
+    pub y_limbs: Limbs<T, <U256Field as NumLimbs>::Limbs>,
+    // output values. x / y = output
+    pub output: FieldOpCols<T, U256Field>,
+    pub output_range_check: FieldLtCols<T, U256Field>,
+}
+
+#[derive(Clone, Debug)]
+#[repr(C)]
+pub struct Uint256InvLayer<WitT> {
+    pub wits: Uint256InvWitCols<WitT>,
+}
+
+#[derive(Clone, Debug)]
+pub struct Uint256InvLayout<E: ExtensionField, Spec: Uint256InvSpec> {
```

Comment:
Implementation refer from `Uint256MulLayout` within same file.
Just simplify to use in Inv case (x = 1, modulus is constant, x_inv = 1 / x) and avoid redundant witness allocation
Besides also generalize to any U256 prime field to work.